spec/controllers/insured/interactive_identity_verifications_controller_spec.rb

```diff
@@ -4,16 +4,18 @@
   describe Insured::InteractiveIdentityVerificationsController do
 
     describe "GET #new" do
-      let(:mock_person) { double(agent?: false) }
+      let(:mock_person) { double(agent?: false, policy_class: PersonPolicy) }
       let(:mock_family) { instance_double("Family") }
       let(:mock_transaction_id) { double }
       let(:mock_user) { double(:person => mock_person) }
       let(:mock_service) { instance_double("::IdentityVerification::InteractiveVerificationService") }
       let(:mock_service_result) { instance_double("::IdentityVerification::InteractiveVerificationResponse", :failed? => service_failed, :to_model => mock_session, :transaction_id => mock_transaction_id) }
       let(:mock_session) { double }
       let(:mock_template_result) { double }
+      let(:mock_policy) { instance_double(PersonPolicy, :complete_ridp? => true) }
```

Comment:
why using a double on the policy ? this will not test the new code

Reply:
The instance double is used here - however note that the test is performed 'for real' in the policy spec.

Inside the controller we only care about 2 cases: when it is allowed and when it isn't.